src/templates/organization.js

```diff
@@ -73,7 +79,8 @@ function Tags({ org }) {
 }
 
 export default function OrganizationTemplate({ data }) {
-  const siteTitle = data.site.siteMetadata.title
+  const { siteTitle } = data.site.siteMetadata
```

Comment:
Is this a regression? Would expect it to be `const { title }` to keep the same value.

Reply:
yep